.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -313,6 +313,12 @@ jobs:
         QT_QPA_PLATFORM: ${{ matrix.qt_qpa_platform }}
         QT_SELECT: qt${{ matrix.qt_version }}
 
+    - name: "tmate session for debugging"
```

Comment:
note this should be removed before merging :-)

Reply:
Done